frontends/web/src/routes/account/receive/receive.jsx

```diff
@@ -65,9 +65,9 @@ export default class Receive extends Component {
     }
 
     verifyAddress = () => {
-        this.setState({ verifying: true });
         const { receiveAddresses, activeIndex } = this.state;
         if (receiveAddresses && activeIndex) {
```

Comment:
Shouldn't we use the same fix here? `if (receiveAddresses !== null && activeIndex !== null)`

Reply:
I could swear I have done that :laughing: Good catch. Done.